### PR TITLE
Populate 2026 and 2027

### PR DESCRIPTION
**Pull request by vishakseshadri**

### CODE REVIEW

File: data/US.yml [hunk 1]
```diff
@@ -232,3 +232,25 @@ US:
...
+  - '2027-01-18'
+  - '2027-02-15'
+  - '2027-05-31'
+  - '2027-06-19'
+  - '2027-07-05'
+  - '2027-09-06'
+  - '2027-10-11'
+  - '2027-11-11'
+  - '2027-11-25'
+  - '2027-12-26'
```
Comment (mkreger1) [inline data/US.yml:256]:
```suggestion
  - '2027-12-24'
  - '2027-12-31'
```

File: data/US.yml [hunk 2]
```diff
@@ -232,3 +232,25 @@ US:
...
+  - '2026-09-07'
+  - '2026-10-12'
+  - '2026-11-11'
+  - '2026-11-26'
+  - '2026-12-25'
+  - '2027-01-01'
+  - '2027-01-18'
+  - '2027-02-15'
+  - '2027-05-31'
+  - '2027-06-19'
```
Comment (mkreger1) [inline data/US.yml:250]:
```suggestion
  - '2027-06-18'
```

File: data/US.yml [hunk 3]
```diff
@@ -232,3 +232,25 @@ US:
   - '2025-11-11'
   - '2025-11-27'
   - '2025-12-25'
+  - '2026-01-01'
+  - '2026-01-19'
+  - '2026-02-16'
+  - '2026-05-25'
+  - '2026-06-19'
+  - '2026-07-04'
```
Comment (mkreger1) [inline data/US.yml:240]:
```suggestion
  - '2026-07-03'
```

**Review by mkreger1 — CHANGES_REQUESTED** [review]
Some of those official days fall on weekends, and the official observance is pushed earlier.

**Comment by vishakseshadri** [comment]
@mkreger1 - The dates come from here - https://www.federalreserve.gov/aboutthefed/k8.htm
If a holiday falls on a Saturday, the previous Friday is NOT a bank holiday, but if a holiday falls on a Sunday, the following Monday IS a bank holiday. July 4th 2026, Jun 19th 2027, Dec 25th 2027 and Jan 1st 2028 all fall on Saturdays, so the previous Fridays are not considered holidays. The date change is applicable only for July 4th, 2027 so 7/5/27 is a holiday

**Comment by mkreger1** [comment]
_I guess_ the Fed is good enough for me. The other websites are probably not accurate.

**Review by mkreger1 — APPROVED** [review]

**Pull request merged**